### ISSUE TITLE
Sessions not reporting in dashboard

### ISSUE THREAD

**Issue by elsasquash** [flurry/react-native-flurry-sdk#17]
Hi! Maybe this does not belong in here, but I'll give it a try.

I did the setup without any problem, but now I see the following issues:
- Events are shown in dashboard with several hours of delay. 
- No session is being shown in the dashboard. **This is what matters the most**

Is there any additional step needed to start reporting sessions?

This is my code 
```js
new Flurry.Builder()
      .withCrashReporting(true)
      .withLogEnabled(true)
      .withLogLevel(Flurry.LogLevel.DEBUG)
      .build(Config.FLURRY_ANDROID_API_KEY, Config.FLURRY_IOS_API_KEY);
```

and then
```js
Flurry.logEvent('React Native Event');
```

**Comment by xnth97** [comment]
Hi @elsasquash ,

Thank you for using Flurry! Sorry to hear that you're having issues with Flurry SDK. There's no additional step required for reporting session as long as you called the `build()` method. Could you please check again and see if your session data is updated correctly? We also have a realtime usage dashboard so that you can see your data with a less delay.

Please let me know if you are still having this issue.

Thanks!

**Comment by poting-oath** [comment]
@elsasquash 5.3.0 just released with the session delay fix. Thanks!

**Comment by poting-oath** [comment]
@elsasquash We have released 5.3.9 for apps use React Native < 0.60, and 5.4.0 for >= 0.60.
Both are based on the same codes and same Flurry SDK libraries. Please try to see whether it fixes your session delay issue or not. Thanks!

**Comment by poting-oath** [comment]
> We have released 5.3.9 for apps use React Native < 0.60, and 5.4.0 for >= 0.60.
Both are based on the same codes and same Flurry SDK libraries. Please try to see whether it fixes your session delay issue or not. Thanks!

Closed
